demos/sjone/light_sensor/makefile

```diff
@@ -0,0 +1,15 @@
+# sjsu_dev2.mk holds the $(SJSU_DEV2_BASE) variable which holds the location of
```

Comment:
What is light sensor code doing in here?

Reply:
I might have done something wrong when squashing my commits